.github/CODEOWNERS

```diff
@@ -30,19 +30,19 @@
 /tests/testdata/modules/object-cache        @tillkruss @spacedmonkey
 
 # Module: WebP Uploads
-/modules/images/webp-uploads                                                @adamsilverstein @felixarntz @mitogh
-/tests/modules/images/webp-uploads                                          @adamsilverstein @felixarntz @mitogh
-/tests/testdata/modules/images/webp-uploads                                 @adamsilverstein @felixarntz @mitogh
+/modules/images/webp-uploads                                                @adamsilverstein @felixarntz
+/tests/modules/images/webp-uploads                                          @adamsilverstein @felixarntz
+/tests/testdata/modules/images/webp-uploads                                 @adamsilverstein @felixarntz
 
 # Module: WebP Support Health Check
-/modules/images/webp-support                                                @adamsilverstein @kirtangajjar @mitogh
-/tests/modules/images/webp-support                                          @adamsilverstein @kirtangajjar @mitogh
-/tests/testdata/modules/images/webp-support                                 @adamsilverstein @kirtangajjar @mitogh
+/modules/images/webp-support                                                @adamsilverstein
+/tests/modules/images/webp-support                                          @adamsilverstein
+/tests/testdata/modules/images/webp-support                                 @adamsilverstein
 
 # Module: Autoloaded Options Health Check
-/modules/database/audit-autoloaded-options                                  @manuelRod
-/tests/modules/database/audit-autoloaded-options                            @manuelRod
-/tests/testdata/modules/database/audit-autoloaded-options                   @manuelRod
+/modules/database/audit-autoloaded-options                                  @manuelRod @felixarntz
+/tests/modules/database/audit-autoloaded-options                            @manuelRod @felixarntz
+/tests/testdata/modules/database/audit-autoloaded-options                   @manuelRod @felixarntz
```

Comment:
There is #889 left to do as a crucial enhancement to this module, so potentially the person who picks up that issue would be a great candidate to include as additional code owner here.